src/Sentry/ISpanTracer.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+
+namespace Sentry
+{
+    /// <summary>
+    /// Span tracer.
+    /// </summary>
+    public interface ISpanTracer : ISpan
+    {
+        /// <summary>
+        /// Span description.
+        /// </summary>
+        // 'new' because it adds a setter.
+        new string? Description { get; set; }
+
+        /// <summary>
+        /// Span operation.
+        /// </summary>
+        // 'new' because it adds a setter.
+        new string Operation { get; set; }
+
+        /// <summary>
+        /// Span status.
+        /// </summary>
+        // 'new' because it adds a setter.
+        new SpanStatus? Status { get; set; }
+
+        /// <summary>
+        /// Starts a child span.
+        /// </summary>
+        ISpanTracer StartChild(string operation);
```

Comment:
You should be able to create a child from a span. Why did this need to move to this interface?

Reply:
Span effectively cannot be modified, as it represents the final state included in a transaction. It's the same as Java's `SentrySpan` vs `Span`.

Reply:
Isn't SpanContext cannot be modified?

Reply:
on Java, `Span` takes a `SentryTracer` as ctor param and when `startChild` is called, it calls `SentryTracer#startChild`